common/src/main/java/com/basistech/rosette/api/common/AbstractRosetteAPI.java

```diff
@@ -45,6 +45,8 @@
     public static final String TOKENS_SERVICE_PATH = "/tokens";
     public static final String SENTENCES_SERVICE_PATH = "/sentences";
     public static final String TEXT_EMBEDDING_SERVICE_PATH = "/text-embedding";
```

Comment:
We should delete the old one right? For any customer who bothers to upgrade to the new version of binding, they should update their calling code to reflect the new endpoints.

Reply:
We certainly can.  I've been leaving the old functionality in place in all the languages assuming we'll just remove it in April when the endpoint goes away.  But yes, since the new binding functionality is opt-in, we could also just remove it now

Reply:
Sure April!